Zadanie 3/novigrad_monsters.html.erb

```diff
@@ -0,0 +1,17 @@
+<table>
```

Comment:
Ten widok i `index.html.erb` są bardzo podobne - można to uprościć używając [partiali](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials) 😉